tests/etl/test_loaders.py

```
@@ -0,0 +1,51 @@
+import pathlib
+import sys
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2]))
+import pandas as pd
+
+from src.etl.loaders import PostgreSQLLoader, TableMappingLoader
+
+
+class FakeDBManager:
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)